### DISCUSSION TITLE
How do you enter retour payments?

### DISCUSSION THREAD

**Discussion by DunklerPhoenix** [firefly-iii/firefly-iii#5943] [Firefly III] [answered]
Heho

How do you register retour/cancellation payments in firefly? Do you create a separate revenue account for that or do you zero-out the original payment ?

Is it possible to mark an account as revenue and expense at the same time or link two accounts together?

**Comment by JC5** [comment, marked as answer]
https://docs.firefly-iii.org/firefly-iii/faq/financials/#how-do-i-handle-partial-refunds

**Reply by DunklerPhoenix** [reply]
Sorry didn't see this ^^'
